openpype/plugins/publish/extract_burnin.py

```diff
@@ -254,6 +254,10 @@ def main_process(self, instance):
             # Add context data burnin_data.
             burnin_data["custom"] = custom_data
 
+            # Add data members.
+            for key in instance.data.get("burninDataMembers", []):
+                burnin_data[key] = instance.data[key]
```

Comment:
Why shouldn't we've just stored the data directly in the `burninDataMembers` as a `dict`. Seems a bit odd to go through `instance.data` here?

Reply:
Yeah, sounds good.